resources/views/faults/app.blade.php

```diff
@@ -0,0 +1,34 @@
+@extends('layouts.app')
+
+@section('content')
+<div class="container">
+    <div class="row justify-content-center">
+        <div class="col-md-8">
+            <div class="card">
```

Comment:
By removing these two line, we could get much more space.

Reply:
Thanks for the suggestion, done.